### ISSUE TITLE
BlockUI does not unblock when blocked property changes fast

### ISSUE THREAD

**Issue by blurbiness** [primefaces/primevue#486]
When the `blocked` property in `BlockUI` component changes fast (it seems that this means faster than the CSS transition), the `unblock` method is called, but the mask is never removed.

Following HTML can be open in a web browser to reproduce the behaviour:

```html
<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>BlockUI unblock issue</title>
    <link href="https://unpkg.com/primevue/resources/themes/mdc-dark-deeppurple/theme.css" rel="stylesheet">
    <link href="https://unpkg.com/primevue/resources/primevue.css " rel="stylesheet">
    <link href="https://unpkg.com/primeflex/primeflex.css" rel="stylesheet">
    <script src="https://unpkg.com/vue@2.6.11/dist/vue.js"></script>
    <script src="https://unpkg.com/primevue/components/panel/panel.umd.js"></script>
    <script src="https://unpkg.com/primevue/components/inputtext/inputtext.umd.js"></script>
    <script src="https://unpkg.com/primevue/components/button/button.umd.js"></script>
    <script src="https://unpkg.com/primevue/components/blockui/blockui.umd.js"></script>
    <style>
        html {
            font-size: 14px;
        }
        body {
            font-family: var(--font-family);
            background-color: var(--surface-a);
            color: var(--text-color);
        }
    </style>
</head>

<body>
<h1>PrimeVue BlockUI issue</h1>

<div id="app">
    <div class="p-input-filled">
        <div>
            <p>If the value bound to BlockUI blocked property changes faster than the CSS transition, BlockUI never unblocks.</p>
            <p>Click on "Slow" button and all works ok. But if you click on "Fast", then BlockUI is blocked forever.</p>
        </div>    
        <p-blockui :blocked="isBusy" @block="debug('BlockUI block event')" @unblock="debug('BlockUI unblock event')">
            <p-panel header="This will block for a while">
                <div class="p-field"><p-inputtext v-model="text"></p-inputtext></div>
                <div class="p-field"><p-button @click="submit(1000)">Slow</p-button></div>
                <div class="p-field"><p-button @click="submit(20)">Fast</p-button></div>
            </p-panel>    
        </p-blockui>
    </div>
    <pre>isBusy = {{isBusy}}</pre>
</div>

<script>
new Vue({
    components: {
        'p-panel': panel,
        'p-button': button,
        'p-inputtext': inputtext,
        'p-blockui': blockui,
    },
    data: function() {
        return {
            text: 'foo',
            isBusy: false
        }
    },
    methods: {
        submit: function(t) {
            this.isBusy = true;
            setTimeout(() => {
                this.isBusy = false;
            }, t);          
        },
        debug: function(msg) {
            console.warn(msg);
        }
    },
}).$mount('#app')
</script>
</body>
```

**Comment by cagataycivici** [comment]
Unable to replicate with the latest PrimeVue, please try with the latest version available and create a new ticket if issue persists so that we can work on it again. Thank you.
